cpp/libjsonnet++.cc

```diff
@@ -0,0 +1,157 @@
+// Copyright 2015 Google Inc. All rights reserved.
```

Comment:
What do you mean? Are you referring to the `cc_library` rules under `include/`? The problem is that since the target for `cpp/libjsonnet++.cc` are in `cpp/BUILD`, directly adding `../include/libjsonnet++.h` would cross package boundaries, and Bazel would complain. The way to include headers in a separate package is to add a `cc_library` rule for the header and then add that rule to `deps`.